services/core-api/app/api/verifiable_credentials/manager.py

```diff
@@ -87,6 +97,87 @@ def offer_newest_amendment_to_current_permittee(permit_amendment_guid: str,
     return info_str
 
 
+def process_all_untp_map_for_orgbook():
+    """Revoke the existing credential and offer a new one with the newest amendment."""
+    # find all parties connected to orgbook
+    # find all most recent permit_amendments for each permit they are permittee's on.
+    # for each permit_amendment, produce the untp_cc_map payload
+
+    # https://metabase-4c2ba9-prod.apps.silver.devops.gov.bc.ca/question/2937-permit-amendments-for-each-party-orgbook-entity
+    permit_amendment_query_results = db.session.execute("""
```

Comment:
It was way easier for me to construct this from Metabase and bring this in. 

I can look at making this object relationships if that is preferred (this SQL query will be much faster though), but performance is not a huge concern.